lib/python/picongpu/plugins/data/base_reader.py

```diff
@@ -23,10 +26,29 @@ def __init__(self, run_directory):
             raise ValueError('The run_directory parameter can not be None!')
 
         self.run_directory = run_directory
+        self.find_time = FindTime(run_directory)
+
         # need to be set in derived classes
         self.data_file_prefix = None
         self.data_file_suffix = None
 
+    def get_dt(self):
```

Comment:
Do we need this as public API for all data readers?
We can do this for convenience, but it might be a bit doubled.